youtube/generic.md

```diff
@@ -1,37 +1,46 @@
-...
-
-I now have a 2nd YouTube channel for short video clips of 60s or less, subscribe ... 
-https://www.youtube.com/eddiejaoudetv
-
-JOIN our inclusive open source community EddieHub now for FREE ... http://eddiehub.org, also checkout my website and sign up to our community NEWSLETTER... http://eddiejaoude.io for my live stream + video schedule and community calls
-
-Also join our community GitHub organisation by creating an issue with the type "invite me to the organisation" :)
-
----------------------------------------------------------------------------------------------------
-
-A BIG thank you to my GitHub Sponsors:
-***
-Stephen Mount http://github.com/stemount
-Nicholas Carrigan https://github.com/nhcarrigan
-***
-
-- Santosh Yadav https://github.com/santoshyadavdev
-- Andrew Cunliffe https://github.com/andrew-cunliffe
-- Alexander Stoichkov https://github.com/SashoStoichkov
-- Allan Regush https://github.com/AllanRegush
-
----------------------------------------------------------------------------------------------------
-
-Don’t forget to support my channel by subscribing below, it’s free, and also share with your friends. Subscribe now!!
-https://www.youtube.com/eddiejaoude?sub_confirmation=1
-
-Follow on other socials for behind the scenes footage, join discord to continue the conversation...
-
-http://eddiejaoude.io 
-https://github.com/eddiejaoude
-http://youtube.com/eddiejaoude
-http://twitter.com/eddiejaoude
-http://instagram.com/eddiejaoude
-http://linkedin.com/in/eddiejaoude
-https://discord.com/invite/jZQs6Wu
-http://github.com/EddieHubCommunity
+## YouTube Channel
+Don’t forget to support my channel by subscribing it! It’s free: [youtube.com/eddiejaoude](https://youtube.com/eddiejaoude?sub_confirmation=1)
```

Comment:
afaik Markdown does **not** work in youtube description.
The entire point of this file is so that Eddie can copy-and-paste the text onto the description section.
```suggestion
Don’t forget to support my channel by subscribing it! It’s free: https://youtube.com/eddiejaoude?sub_confirmation=1
```
It would be great if we could avoid using specific-markdown(like hyperlinks), wherever possible.